### ISSUE TITLE
eslint --fix not being executed at all

### ISSUE THREAD

**Issue by James2516** [prettier/prettier-eslint#160]
**Versions**:

- `prettier-eslint` version: 8.3.1
- `node` version: 8.6.0
- `npm` (or `yarn`) version: yarn 1.3.2
 
**Have you followed [the debugging tips](https://github.com/prettier/prettier-eslint/blob/master/README.md#debugging-issues)?**

Yes
**Relevant code or config**

```
{
  "root": true,
  "extends": "airbnb",
  "parserOptions": {
    "ecmaVersion": 6,
    "sourceType": "module"
  },
  "env": {
      "es6": true
  },
  "parser": "babel-eslint",
  "rules": {
    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
    "react/prop-types": 0,
    "no-console": 0,
    "arrow-parens": ["error", "always"],
    "prefer-template": "error"
  }
}
```

.prettierrc:
```
{
  "singleQuote": true,
  "trailingComma": "all",
  "arrowParens": "true"
}
```

**What I did:**

![](https://i.imgur.com/fgcqM31.png)

I have eslint integration enabled on `prettier-atom`, but it doesn't seem to be running `eslint --fix` at all.

If I enable `linter-eslint`'s Fix errors on save, it'd automatically insert a line after the `import` statement to fix that warning.

However I'm not supposed to do that as it gives a warning:
> prettier-atom: linter-eslint's Fix on Save feature is currently enabled. We recommend disabling this feature when using the ESlint integration from prettier-atom

**Comment by zimme** [comment]
Maybe there's a problem with this logic, which decides if only prettier should be run. https://github.com/prettier/prettier-eslint/blob/master/src/index.js#L99-L117

**Comment by zimme** [comment]
Fixed in 8.7.6, please test with that version.

I'll notify the maintainers of the `prettier-atom` and `prettier-vscode` to update their version of `prettier-eslint`
